components/constellation/constellation.rs

```diff
@@ -2733,6 +2740,14 @@ where
         if let Err(e) = storage_ipc_receiver.recv() {
             warn!("Exit storage thread failed ({:?})", e);
         }
+        match (self.webgl_threads.take(), webgl_threads_receiver) {
```

Comment:
I would do `if.webgl_threads.is_some()` and expect ` webgl_threads_receiver` to be some inside the block.